.github/workflows/pull-transifex.yml

```diff
@@ -31,6 +31,33 @@ jobs:
           beautifyJSON "ownCloudSDK/Resources/Localizable.xcstrings"
           beautifyJSON "ownCloudUI/Resources/Localizable.xcstrings"         
 
+      - name: Install dependencies
```

Comment:
Isn't there any pre-built docker image with swift which can be used? Installing swift to compile a build steps feels wrong.
Default GitHub images come with so many pre-installed interpreters - there must be an easier solution ....

Reply:
You're right. Upon looking into it, I quickly discovered a GH action for setting up Swift and updated the yml accordingly. Please have another look.